superduperdb/backends/ibis/query.py

```diff
@@ -815,7 +824,12 @@ def compile(
             return getattr(parent, self.name), tables
         args, tables = _compile_item(self.args, db, tables=tables)
         kwargs, tables = _compile_item(self.kwargs, db, tables=tables)
-        return getattr(parent, self.name)(*args, **kwargs), tables
+        try:
+            return getattr(parent, self.name)(*args, **kwargs), tables
+        except AttributeError:
+            raise QueryValidationError(
+                '{} is not a valid query member of ibis query'.format(self.name)
```

Comment:
It would be a bit shorter if you use an [f-string](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) here.
```suggestion
                f'{self.name} is not a valid query member of an ibis query'
```

Reply:
yes